src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,36 @@
+.form-field {
+  display: block;
+  width: 100%;
+  height: 48px;
+  padding: 14px 16px;
+
+  color: $c-main;
+
+  background-color: #f8f8fa;
+  border: 1px solid transparent;
+  border-radius: 40px;
```

Comment:
it's too much, as for me 🙃

<img width="571" alt="image" src="https://user-images.githubusercontent.com/55802630/174539909-39e0a2b1-0632-4c1f-9544-0befcb38a058.png">